README.md

```diff
@@ -206,15 +206,48 @@ Read the important message after you run `scxa-solrlcoud-bootstrap`:
 > takes a few extra minutes: the exception is thrown before the process has completed.
 > The best option is to manually build and supervise this step.
 >
-> Run the following command (don’t worry if the request returns a 500 error after a while):
+> Run the following commands one by one(don’t worry if the request returns a 500 error after a while):
 > 
 > `docker exec -i scxa-solrcloud-1 curl 'http://localhost:8983/solr/bioentities-v1/suggest?suggest.build=true&suggest.dictionary=propertySuggesterNoHighlight'`
+
+> `docker exec -i scxa-solrcloud-1 curl 'http://localhost:8983/solr/bioentities-v1/suggest?suggest.build=true&suggest.dictionary=bioentitySuggester'`
 >
 > While the command above is running, monitor the size of the suggester directory size:
 > 
 > `docker exec -it scxa-solrcloud-1 bash -c 'watch du -sc server/solr/bioentities-v1*/data/*'`
 > `docker exec -it scxa-solrcloud-2 bash -c 'watch du -sc server/solr/bioentities-v1*/data/*'`
 >
-> The suggester in all shards will be built when the propertySuggester directory size stabilises. 
-> Run the above procedure for `bioentitySuggester` as well.
- 
+> The suggester in all shards will be built when the propertySuggester directory size stabilises.
+
+### Run Tests using Gradle container
+If you invoke docker-compose-gradle.yml , it will spin up Solr and Zookeeper containers, creates a gradle container and run tests in the Gradle container.
+If you are already running the scxa application using docker-compose, if you invoke the docker-compose-gradle.yml file,it uses already running Solr and Zookeeper containers to run tests.
+
+#### Note: A few tests depends on suggesters. So don't forget to build suggestions in your local machine.
+
+In the `atlas-web-single-cell/docker` directory run the following:
+
+```bash
+ATLAS_DATA_PATH=/path/to/sc/atlas/data \
+GRADLE_HOST=scxa-gradle\
+PWD=/path/to/sc/project \
+ATLAS_MAVEN_CACHE=/path/to/maven/cache \
+ATLAS_GRADLE_CACHE=/path/to/gradle/cache \
+docker-compose -f docker-compose-test.yml up
```

Comment:
I think you mean `docker-compose-gradle` instead of `docker-compose-test` here.

Reply:
yes, sorry, copy paster error :)

Reply:
Fixed